src/tests/junit/TemperatureConverterTests.java

```diff
@@ -0,0 +1,17 @@
+package tests.junit;
+
+import converter.TemperatureConverter;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TemperatureConverterTests {
+    @Test
+    public void firstTest() throws Exception {
+        TemperatureConverter uut = new TemperatureConverter();
```

Comment:
`sut`? I've never seen `uut`

used widely in https://www.amazon.com/xUnit-Test-Patterns-Refactoring-Code-dp-0131495054/dp/0131495054

Reply:
```
   @Test
    public void fToC_HappyPath() throws Exception {
        TemperatureConverter sut= new TemperatureConverter();
        String expected = "from F => C -17.77777777777778";

        String actual =  sut.convert("0 F");
        
        assertThat(actual).isEqualTo(expected);
    }
```
compare with yours, I prefer a more pragmatic approach. 

Note: in the assert I propose to use assertJ
https://joel-costigliola.github.io/assertj/

Reply:
ca=hanged assertions from jUnit assertions to asertj